pages/api/v1/progressReport/addReportRating.ts

```
@@ -0,0 +1,74 @@
+		}
+
+		const user = await authServiceInstance.GetUser(token);
+		if (!user) {
+			return res.status(403).json({ message: messages.UNAUTHORISED });
+		}
+
+		const user_id = user?.id?.toString();
+
+		const { postId, proposalType, rating } = req.body;
+
+		if (!proposalType || rating === undefined || isNaN(postId) || !isProposalTypeValid(proposalType)) {
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)